pkg/virt-controller/services/template_test.go

```diff
@@ -465,10 +465,13 @@ var _ = Describe("Template", func() {
 				Expect(pod.Spec.SecurityContext.SELinuxOptions).ToNot(BeNil())
 				Expect(pod.Spec.SecurityContext.SELinuxOptions.Type).To(Equal("spc_t"))
 			})
-			It("should have a level of s0 on all but compute if a type is specified", func() {
+			DescribeTable("should have an SELinux level of", func(enableWorkaround bool) {
 				config, kvInformer, svc = configFactory(defaultArch)
 				kvConfig := kv.DeepCopy()
 				kvConfig.Spec.Configuration.SELinuxLauncherType = "spc_t"
+				if enableWorkaround {
```

Comment:
Can you please append the selinux FG? It may be that the current FG list isn't empty.
I think we have a helper function somewhere in test framework that does it.

Reply:
@jean-edouard Please ignore, its unit tests.

Reply:
This might be good practice actually.